jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -173,9 +111,13 @@ void setDisableUpdateCheck(boolean disableUpdateCheck) {
                     .build());
         VersionJsonTemplate version =
             JsonTemplateMapper.readJson(response.getBody(), VersionJsonTemplate.class);
-        Path lastUpdateCheckTemp = configDir.resolve(LAST_UPDATE_CHECK_FILENAME + ".tmp");
+
+        Path lastUpdateCheckTemp =
```

Comment:
should we then instead have a tmp dir in here? or does that not matter?

Reply:
You mean create like `<config dir>/some tmp dir/lastUpdateCheck.tmp`? I think it's unnecessary if we rename it to `<config dir>/lastUpdateCheck`. Also, renaming a file within the same directory feels safer than moving from a different directory.